src/OperatingSystemApp/toolbox.js

```diff
@@ -14,6 +14,7 @@ var OSToolbox = GObject.registerClass(class OSToolbox extends Toolbox {
 
     bindWindow(win) {
         void (this, win);
```

Comment:
I think you can remove this line now

Reply:
unfortunately I can not, because it complains the function should use the this pointer :S